### PR TITLE
Utilize `nameof` operator

### PR DESCRIPTION
**Pull request by henkmollema**
Utilized the `nameof` operator with `ArgumentExceptions` in the following projects:
- `Microsoft.AspNet.Mvc.Core`
- `Microsoft.AspNet.Mvc.ModelBinding`
- `Microsoft.AspNet.Mvc.Razor.Host`
- `Microsoft.AspNet.Mvc.WebApiCompatShim`

### CODE REVIEW

File: src/Microsoft.AspNet.Mvc.Core/AntiForgery/AntiForgeryTokenSet.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ public AntiForgeryTokenSet(string formToken, string cookieToken)
         {
             if (string.IsNullOrEmpty(formToken))
             {
-                throw new ArgumentException(Resources.ArgumentCannotBeNullOrEmpty, formToken);
+                throw new ArgumentException(Resources.ArgumentCannotBeNullOrEmpty, nameof(formToken));
```
Comment (henkmollema) [inline src/Microsoft.AspNet.Mvc.Core/AntiForgery/AntiForgeryTokenSet.cs]: Looks like this was a mistake.

Comment (Eilon) [inline src/Microsoft.AspNet.Mvc.Core/AntiForgery/AntiForgeryTokenSet.cs]: Yup, good find.

**Comment by Eilon** [comment]
@kichalla can you take a look and merge?

**Comment by pranavkm** [comment]
:shipit:

File: src/Microsoft.AspNet.Mvc.Core/FormatterMappings.cs [hunk 2]
```diff
@@ -68,7 +68,7 @@ private string RemovePeriodIfPresent(string format)
         {
             if (format == "")
```
Comment (pranavkm) [inline src/Microsoft.AspNet.Mvc.Core/FormatterMappings.cs]: This should probably be `string.IsNullOrEmpty`

Comment (henkmollema) [inline src/Microsoft.AspNet.Mvc.Core/FormatterMappings.cs]: Should that change be in this commit too? Seems a bit out of scope.

Comment (Eilon) [inline src/Microsoft.AspNet.Mvc.Core/FormatterMappings.cs]: Yeah separate change.

Comment (kichalla) [inline src/Microsoft.AspNet.Mvc.Core/FormatterMappings.cs]: I will then go ahead and merge this PR.

**Comment by kichalla** [comment]
Merged: https://github.com/aspnet/Mvc/commit/92554fa634b9feea1c3ad6688e83c598f8875aca
